docs/source/how-to/deploying-substra/upgrade-notes.rst

```diff
@@ -4,6 +4,93 @@
 Upgrade notes
 *************
 
+.. _ops upgrade notes 0.34:
+
+Substra 0.34.0
+--------------
+
+This version upgrades the integrated PostgreSQL databases from versions 14 to 16. This upgrade must be performed manually.
+
+If you :ref:`do not use the integrated databases <ops howto external database>`, then you have nothing to do.
+
+Otherwise, you must update the data format. There are many ways to do this; here we give you a simple way which is acceptable if you have small amounts of data, you have access to the clusters, and downtime is acceptable. It must be repeated on each Substra node and orchestrator.
+
+#. Make sure no operations are in progress and cordon the node.
+
+#. Gather required info:
+   * database credentials (here ``$PG_USER`` and an associated password)
+   * Kubernetes namespace (here ``$NAMESPACE``)
+   * Helm release (hereafter ``$RELEASE_NAME``, can be obtained from ``helm get release -A``)
+   * hostname of the Postgres service (hereafter ``$HOST``, can be obtained from ``kubectl get svc -n $NAMESPACE``)
+   * name of the database in Postgres: by default ``orchestrator`` for the orchestrator, and ``substra`` for the backend (hereafter ``$DB``)
+
+#. Deploy a psql client (we will call ``postgres-backup``) in the namespace
+
+   .. code-block:: bash
+
+      kubectl apply -f -n $NAMESPACE - << 'EOF'
+      apiVersion: v1
+      kind: Pod
+      metadata:
+        name: postgres-backup
+      spec:
+        containers:
+          - name: postgres
+            image: postgres
+            command: ["sleep", "infinity"]
+      EOF
+
+#. Get a shell **onto the postgres-backup pod**, dump the DB:
+
+   .. code-block:: bash
+
+      pg_dump $DB --host=$HOST -U $PG_USER -f /dump.sql --clean --create
+
+#. (OPTIONAL) Retrieve the dump:
+
+   .. code-block:: bash
+
+      kubectl --retries 10 cp $NAMESPACE/postgres-backup:/dump.sql dump.sql
+      sha1sum dump.sql # check it matches with the source (use `shasum` on Mac OS)
+
+
+#. Delete the Postgres StatefulSet and PVC
+
+   This depends on your particular set-up but it should look like this:
+
+   .. code-block:: bash
+
+      kubectl delete -n substra statefulset $RELEASE_NAME-postgresql
+      kubectl delete -n substra pvc data-$RELEASE_NAME-postgresql-0
+
+#. Perform the database upgrade
+
+   Note versions ``8.0.0`` and ``23.0.0`` are used: they contain the database upgrade but not the app upgrade.
```

Comment:
I'm afraid I do not understand the sentence here.

Reply:
apps upgrade are in 8.0.1 and 23.0.2

Reply:
Could we rephrase it to make it a bit clearer?